client/jetpack-cloud/sections/partner-portal/issue-license-v2/index.tsx

```diff
@@ -77,6 +77,29 @@ export default function IssueLicenseV2( { selectedSite, suggestedProduct }: Assi
 			? translate( 'Single license' )
 			: ( translate( '%(size)d licenses', { args: { size: selectedSize } } ) as string );
 
+	const selectedCount = selectedLicenses.filter( ( license ) => license.quantity === selectedSize )
+		?.length;
+
+	const navItems = availableSizes.map( ( size ) => {
+		const count = selectedLicenses.filter( ( license ) => license.quantity === size ).length;
+		return {
+			label:
+				size === 1
+					? translate( 'Single license' )
+					: ( translate( '%(size)d licenses', {
+							args: { size },
+					  } ) as string ),
+			selected: selectedSize === size,
+			onClick: () => setSelectedSize( size ),
+			...( count && { count } ),
```

Comment:
[non-blocker] I find `count: count ? count : undefined` more readable. It seems unusual to me to pass a single optional prop like this. This is probably my preference.

Reply:
The reason I kept it like this is if the count is passed as a prop, it would display the count as 0.